src/main/java/seedu/address/logic/parser/ViewCommandParser.java

```diff
@@ -36,6 +38,11 @@ public ViewCommand parse(String arg) throws ParseException {
         }
 
         Date date = ParserUtil.parseDate(argMultimap.getValue(PREFIX_DATE).get());
+
+        if (DateTimeUtil.isExpiredByDay(date.getDate())) {
+            throw new ParseException(MESSAGE_EXPIRED_DATE);
+        }
```

Comment:
Would it be better to give a more direct message rather than a generic one? (Since this msg is only used once) 

Maybe we can use: "Cannot view appointments from the past. Past appointments are archived and can be accessed from the archive folder" ?

Let me know what you think! 😄